src/document-click.js

```diff
@@ -131,9 +131,17 @@ export function onDocumentElementClick_(e, viewport, history) {
   // to the custom protocol href.
   const isSafariIOS = platform.isIos() && platform.isSafari();
   const isEmbedded = win.parent && win.parent != win;
```

Comment:
`isEmbedded` is kind of a mute point - this handler is not run when it's not embedded.